src/localstack.ts

```diff
@@ -411,9 +420,9 @@ export async function getConnection <Service extends keyof LocalStackServices>({
     mappedServices,
     getOutput: () => output.toString(),
     clearOutput: () => output.reset(),
-    cleanup: () => {
+    cleanup: async () => {
```

Comment:
Curious, why this change? In most of our projects the linter would complain that this is creating unneeded Promises

Reply:
I changed this while hunting down containers that didn't actually get stopped after all the tests finished.